spec/lib/reports/customers_report_spec.rb

```diff
@@ -96,20 +97,22 @@ module Customers
                 create(:order_with_totals_and_distribution, :completed, distributor: d, bill_address: a, shipping_method: sm)
               }
               before do
+                o1.update(completed_at: Time.zone.yesterday)
+                o2.update(completed_at: Time.zone.today)
                 [o1, o2].each do |order|
                   order.update!(email: "test@test.com")
                 end
               end
 
-              it "returns only one row per customer and count the number of orders" do
+              it "returns only one row per customer with the right data" do
                 expect(subject.query_result).to match_array [[o1, o2]]
                 expect(subject.table_rows.size).to eq(1)
                 expect(subject.table_rows).to eq([[
                                                    a.firstname, a.lastname,
                                                    [a.address1, a.address2, a.city].join(" "),
                                                    o1.email, a.phone, d.name,
                                                    [d.address.address1, d.address.address2, d.address.city].join(" "),
-                                                   o1.shipping_method.name, 2, o1.total + o2.total
+                                                   o1.shipping_method.name, 2, o1.total + o2.total, o2.completed_at.strftime("%Y-%m-%d")
```

Comment:
I would prefer plain text here.

Reply:
Not sure to understand exactly why?

Reply:
> Not sure to understand exactly why?

We came to [this convention](https://github.com/openfoodfoundation/openfoodnetwork/wiki/Code-Conventions#prefer-plain-text-over-method-calls-in-expected-values) for several reasons:

The code `strftime("%Y-%m-%d")` is the same in production code and in the spec. If it contains a mistake, for example using the wrong letter, then we don't notice this in the spec. The reader may also be left wondering how this formatting looks like exactly. Plain text is easier to read.

Reply:
Thanks! Modified ;)